utils/env.go

```diff
@@ -30,6 +31,31 @@ const (
 
 	// AndroidFilesDir is hardcoded because golang inits before our android code can override HOME var.
 	AndroidFilesDir = "/data/user/0/com.viam.rdk.fgservice/cache"
+
+	// ViamEnvVarPrefix is the prefix for all Viam-related environment variables.
+	ViamEnvVarPrefix = "VIAM_"
```

Comment:
Yeah new env vars should definitely start with `VIAM_` now; I feel like that was already an expectation, but thanks for encoding more formally.